version.json

```diff
@@ -0,0 +1,17 @@
+{
+  "version": "5.3",
+  "publicReleaseRefSpec": [
+    "^refs/heads/master$", // we release out of master
+    "^refs/heads/develop$", // we release out of develop
```

Comment:
Please drop last two as this repo has never been released from branches other master